### DISCUSSION TITLE
Please release Microsoft.CodeAnalysis.CSharp.Workspaces 4.5.0!

### DISCUSSION THREAD

**Discussion by chrischu** [dotnet/roslyn#67035] [General] [answered]
Hi!

The new VisualStudio version (17.5) apparently uses a new version of Microsoft.CodeAnalysis.CSharp.Workspaces (namely 4.5.0), but this version is not available on NuGet yet (except as a preview that might not match exactly?),

Could you please release the Microsoft.CodeAnalysis.CSharp.Workspaces in version 4.5.0?

Thanks!

**Comment by jasonmalinowski** [comment, marked as answer]
We are working on it -- some dependencies we depend on are still getting their packages up.

**Comment by pinkfloydx33** [comment]
> new VisualStudio version (17.5) apparently uses

Is there a list somewhere mapping Microsoft.CodeAnalysis.Csharp.Workspaces and Microsoft.CodeAnalysis.Csharp to the corresponding VS versions? I swear there used to be one that listed out the various versions, but I cannot seem to locate it. I'd like to update some of my analyzers but unsure how high I can go...

**Reply by jasonmalinowski** [reply]
You're probably thinking of https://learn.microsoft.com/en-us/visualstudio/extensibility/roslyn-version-support?view=vs-2022. But in general, just add/subtract 13 to go from one to the other. (There's some special cases in VS2017, but it's unlikely at this point you're targeting a special version of that other than "latest".)
